catalog/transformer/catalog-transformer-tika-input/src/test/resources/testpackage/testJAVA.java

```diff
@@ -15,7 +15,6 @@
 
 public class HelloWorld {
```

Comment:
This is used for testJavaSource in TikaInputTransformerTest but failed a checkstyle rule for using System.out.println so I removed that line.